src/crc32.c

```diff
@@ -6,7 +6,6 @@
 //
 
 #include "crc32.h"
-#include <memory.h>
```

Comment:
This looks obsolete to me, but I'm note sure for other platforms

Reply:
Once @wolfmcnally answers this, I'll merge as all the other changes are Arduino specific.